Dockerfile

```
@@ -495,6 +493,54 @@ RUN \
 
 # =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
+FROM sdk-go as sdk-govc
+
+USER root
+RUN \
+  mkdir -p /usr/libexec/tools /usr/share/licenses/govmomi && \
+  chown -R builder:builder /usr/libexec/tools /usr/share/licenses/govmomi
+
+ARG ARCH
+ARG TARGET="${ARCH}-bottlerocket-linux-gnu"
+ARG GOVMOMIVER="0.26.0"
+ARG GOVMOMISHORTCOMMIT="34586b6"
+
+USER builder
+WORKDIR ${GOPATH}/src/github.com/vmware/govmomi
+COPY ./hashes/govmomi /home/builder/hashes
+RUN \
+  sdk-fetch /home/builder/hashes && \
+  tar --strip-components=1 -xf govmomi-${GOVMOMIVER}.tar.gz && \
+  rm govmomi-${GOVMOMIVER}.tar.gz
+
+ARG GOARCH_aarch64="arm64"
+ARG GOARCH_x86_64="amd64"
+ARG GOARCH_ARCH="GOARCH_${ARCH}"
+
+COPY --chown=0:0 --from=sdk-license-scan /usr/libexec/tools/ /usr/libexec/tools/
+COPY --chown=0:0 --from=sdk-license-scan /home/builder/license-scan/clarify.toml .
+RUN \
+  cp -p LICENSE.txt /usr/share/licenses/govmomi && \
+  go mod vendor && \
+  /usr/libexec/tools/bottlerocket-license-scan \
+    --clarify clarify.toml \
+    --spdx-data /usr/libexec/tools/spdx-data \
+    --out-dir /usr/share/licenses/govmomi \
+    go-vendor vendor
+
+WORKDIR ${GOPATH}/src/github.com/vmware/govmomi/govc
+RUN \
+  export GOARCH="${!GOARCH_ARCH}" ; \
+  export CGO_ENABLED=0 ; \
+  export BUILD_VERSION_PKG="github.com/vmware/govmomi/govc/flags" ; \
+  go build -mod=vendor -o /usr/libexec/tools/govc -ldflags " \
+    -X ${BUILD_VERSION_PKG}.BuildVersion=${GOVMOMIVER} \
+    -X ${BUILD_VERSION_PKG}.BuildCommit=${GOVMOMISHORTCOMMIT} \
+    -X ${BUILD_VERSION_PKG}.BuildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
+    "
```

nit: I'd prefer to see the targeted package to be compiled. Especially since we're reconstituting a `GOPATH` this could be the full pkg path to the final bin's `main`:

```suggestion
  go build -mod=vendor -o /usr/libexec/tools/govc -ldflags " \
    -X ${BUILD_VERSION_PKG}.BuildVersion=${GOVMOMIVER} \
    -X ${BUILD_VERSION_PKG}.BuildCommit=${GOVMOMISHORTCOMMIT} \
    -X ${BUILD_VERSION_PKG}.BuildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
    " github.com/vmware/govmomi/govc
```

---

If we go with this, we probably don't have to change workdirs either.